CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 * [#187](https://github.com/slack-ruby/slack-ruby-client/pull/187): Concatenate error message when multiple errors present - [@chrislopresto](https://github.com/chrislopresto).
 * [#188](https://github.com/slack-ruby/slack-ruby-client/pull/188): Fixed `NoMethodError` when Slack is unavailable - [@sonicdoe](https://github.com/sonicdoe).
+* [#196](https://github.com/slack-ruby/slack-ruby-client/pull/196): Update API methods from slack-api-ref - [@manuelmeurer](https://github.com/manuelmeurer).
```

Comment:
For consistency we list the actual API changes. This should say "added lookupByEmail`, etc., see below for examples.

Reply:
Ok, will change that and also squash both commits.